Player/Player1.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+
+</Project>

```

Comment:
Move all Player logic to CourceApp project (may place code in the subfolder) Or - add stylecop and test's connection to the project